### ISSUE TITLE
possible to transmit the message ?

### ISSUE THREAD

**Issue by DavidIgor** [xMasterX/flipper-pager#10]
Hi, wondering if it's possible to transmit messages as well?

**Comment by Nhscan** [comment]
Yeah be nice if we can get transmit I can transmit pocsag from my hackrf porterpack. Hopefully somebody will add it or make it separate have to transmit
